src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,38 @@
+.catalog {
+  &__header {
+    padding-left: 50px;
+    padding-right: 50px;
+    padding-bottom: 1px;
+    background: #fff;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  }
+
+  &__body {
+    justify-content: center;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    padding-top: 50px;
+    padding-left: 40px;
+    padding-right: 40px;
+    padding-bottom: 73px;
+
+    @include onDesktop {
+      padding-left: 128px;
+      padding-right: 128px;
+    }
+  }
```

Comment:
add max-with for tis block

Reply:
put main.scss file to styles directory

Reply:
giving a body tag class is not a very good idea I think. You can split the header and body into different files.